hack/README.md

```diff
@@ -77,27 +78,62 @@ These may be passed to a target as `$ make VARIABLE=value target`
 - `RELREF`: Required by `release-description`. Must be a commit or tag.  Should be the more recent than `PREREF`
 - `PREREF`: Required by `release-description`. Must also be a commit or tag.  Should be the later than `RELREF`
 
-#### Execute Functional Tests
+#### Execute Standard Environment Functional Tests
+
+If using a standard bare-metal/local laptop rhel/kvm environment where nested
+virtualization is supported then the standard *kubevirtci framework* can be used.
+
 Environment Variables and Supported Values
 
 | Env Variable       | Default       | Additional Values  |
 |--------------------|---------------|--------------------|
 |KUBEVIRT_PROVIDER   | k8s-1.10.4    | os-3.10.0          |
 |NUM_NODES           | 1             | 2-5                |
 
-To Run Tests
+To Run Standard *cluster-up/kubevirtci* Tests
 ```
- make cluster-up
- make cluster-sync
- make test-functional
+ # make cluster-up
+ # make cluster-sync
+ # make test-functional
 ```
 
 Clean Up
 ```
- make cluster-down
+ # make cluster-down
 ```
 
-**End to End Functional Tests currently only run on bare-metal - they will not run in a VM/Cloud environment (i.e. GCE, AWS, etc...)**
+#### Execute Alternative Environment Functional Tests
+
+If running in a non-standard environment such as Mac or Cloud where the *kubevirtci framework* is
+not supported, then you can follow the following example to run Functional Tests.
+
+1. Stand-up a Kubernetes cluster (local-up-cluster.sh/kubeadm/minikube/etc...)
+
+2. Clone or get the kubevirt/containerized-data-importer repo
+
+3. Run the CDI controller manifests
```

Comment:
A note about building the file host should be included also as it is/will be required for some of the functional tests.  I've got pr #394 up now to add a `make docker-functest-image` target